Gemfile.lock

```diff
@@ -27,47 +27,31 @@ PATH
 GEM
```

Comment:
I see the point, and for sure we need control over dependency versions. But we can specify version numbers in the Gemfile right?

I mean the Gemfile is the source for getting Gemfile.lock, and as long as the source is stable, then the generated lock file should always be the same.